base_deterministic_session_gc/http.py

```diff
@@ -27,7 +28,7 @@ def deterministic_session_gc(session_store, session_expiry_delay=None):
         "Deleting all sessions inactive since %s",
         datetime.fromtimestamp(expired_time).strftime(DEFAULT_SERVER_DATETIME_FORMAT),
     )
-    for fname in os.listdir(session_store.path):
+    for fname in glob.iglob(os.path.join(session_store.path, "*", "*")):
```

Comment:
```suggestion
    for fname in glob.iglob(os.path.join(session_store.path, "*")):
```

Because in 14, all sessions are in the same directory?

Reply:
Makes sense!